dashboard/app/controllers/projects_controller.rb

```diff
@@ -376,6 +376,28 @@ def remix
     redirect_to action: 'edit', channel_id: new_channel_id
   end
 
+  def export_create_channel
+    return if redirect_under_13_without_tos_teacher(@level)
+    src_channel_id = params[:channel_id]
+    begin
+      _, remix_parent_id = storage_decrypt_channel_id(src_channel_id)
+    rescue ArgumentError, OpenSSL::Cipher::CipherError
+      return head :bad_request
+    end
+    storage_app = StorageApps.new(storage_id('user'))
+    src_data = storage_app.get(src_channel_id)
+    data = initial_data
+    data['name'] = "Exported: #{src_data['name']}"
+    new_channel_id = ChannelToken.create_channel(
+      request.ip,
+      storage_app,
+      data: data,
+      type: params[:key],
+      remix_parent_id: remix_parent_id,
```

Comment:
FYI @poorvasingal and @bencodeorg that an exported project will have the same "remix parent" as the project it was exported from